src/I18n/En.elm

```diff
@@ -169,6 +176,31 @@ enStrings key =
         WhoSubHeading ->
             "Who"
 
+        ---
+        -- SubmitStory Page
+        ---
+        SubmitStoryHeading ->
```

Comment:
We might want to add this and the Index content to an markdown page in the CMS. I'll add to backlog. The intent for `I18n` keys and copy is strictly UI. It feels like we are straying into actual content here - that needs upkeep.

Reply:
oops I miss read this and the markdown already.